src/styles.ts

```diff
@@ -191,6 +191,17 @@ export const commonArticleStyles = ({ kicker }: PillarStyles): SerializedStyles
         }
     }
 
+    .section__rule {
```

Comment:
Are we meant to be using BEM in some places and I haven't realised? 😬

Reply:
Nope, just copied that class from the templates and dotcom.. but let's make it consistently not BEM :)